.github/workflows/test_comprehensive.yml

```diff
@@ -165,3 +163,26 @@ jobs:
         with:
           filename: .github/TEST_FAIL_TEMPLATE.md
           update_existing: true
+
+  test_pip_install:
+    name: Test pip install
```

Comment:
```suggestion
    name: ubuntu-18.04 3.8 pip install
```
Can the name be something more like this for consistency?

![image](https://user-images.githubusercontent.com/6531703/104617351-33176900-5640-11eb-9bd5-c57b92bd1b71.png)